### ISSUE TITLE
api.user.get_roles() not working for Anonymous, throws UserNotFoundError

### ISSUE THREAD

**Issue by AnneGilles** [plone/plone.api#339]
found this while writing a browserview to debug roles of users, checking with an "anonymous" not logged-in user:
```
(Pdb++) api.user.get_roles()
*** UserNotFoundError: 
(Pdb++) api.user.get_current().roles
('Anonymous',)
(Pdb++) api.user.get_current().getRoles()
('Anonymous',)
```

**Comment by fulv** [comment]
Not sure if it's the same issue, but I keep getting this UserNotFoundError, as well.  In my case, however, it is a logged in user.

**Comment by AnneGilles** [comment]
oh, btw, this was Plone 5.1a2 with plone.api 1.5.1

**Comment by jaroel** [comment]
I've fixed this in #341. This will be available in the next plone.api release.
Thank you for reporting!
